modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -374,65 +374,69 @@ gcloud projects add-iam-policy-binding <service-project-id> \
-+
-IMPORTANT: On the *Network* page, select the *Customer-managed* network connection type, and enter the network, service account, and storage bucket information you created. However, before running the Validation `rpk` command, note the `redpanda-id` (for example, `cisld88gfi809ee1qjcg`). You finish the cluster deployment in a later step.  
-+
-. Use the `redpanda-id` to bind service accounts with the Workload Identity role.
-
-* Redpanda cluster service account
+* Redpanda Console service account
 +
 .Show command
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-cluster@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-id>]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<console-gcp-sa-account-id>]"
 ```
-==== 
-* Redpanda Console service account
+====
+
+* Redpanda Connectors service account
 +
 .Show command
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-console@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<redpanda-id>]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<connectors-gcp-sa-account-id>]"
 ```
 ====
 
-* Redpanda Connectors service account
+* Cert-manager and external-DNS service accounts
 +
-.Show command
+.Show commands
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-connectors@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<redpanda-id>]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
 ```
 ====
 
-== Validate and deploy
+== Create a BYOC cluster with customer-managed VPC
+
+Log in to the https://cloudv2.redpanda.com[Redpanda Cloud UI^], and follow the steps to xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster], with the following exceptions:
```

Comment:
Is it worth single-sourcing those instructions rather than sending users off to a separate page?

Reply:
@JakeSCahill I plan to update and single source parts of this BYOC section when we add this same functionality for AWS!